pkg/policy/api/ingress.go

```diff
@@ -116,6 +116,17 @@ type IngressRule struct {
 	//
 	// +kubebuilder:validation:Optional
 	ToPorts PortRules `json:"toPorts,omitempty"`
+
+	// ICMP is a list of ICMP rule identified by type number
+	// which the endpoint subject to the rule is allowed to
+	// receive connections on.
+	//
+	// Example:
+	// Any endpoint with the label "app=httpd" can only accept incoming
+	// type 8 ICMP connections.
+	//
+	// +kubebuilder:validation:Optional
+	ICMP ICMPRules `json:"icmps,omitempty"`
```

Comment:
If the JSON name is going to be plural, then let's make the field name also plural.
```suggestion
	ICMPs ICMPRules `json:"icmps,omitempty"`
```